tests/testthat/test-broom-lda.R

```diff
@@ -0,0 +1,68 @@
+context("broom-lda")
+
+test_that("lda.tidy() works", {
+  sc <- testthat_spark_connection()
+  test_requires_version("2.0.0")
+
+  samples <-  data.frame(text = c("The cat sat on the mat.",
+                                  "The dog ate my homework."))
+
+  lines_tbl <- sdf_copy_to(sc,
+                         samples,
+                         name = "lines_tbl",
+                         overwrite = TRUE)
+
+  td1 <- lines_tbl %>%
+    ml_lda(~text, k = 3) %>%
+    tidy()
+
+  check_tidy(td1, exp.row = 3, exp.col = 4,
+             exp.names = c("topic", "termIndices",
+                           "termWeights", "doc_concentrarion"))
```

Comment:
Is there precedence in these names? Could we just use snake case? Also, typo in `doc_concentration`.